src/devices/Bmp180/Bmp180.cs

```diff
@@ -91,36 +92,39 @@ public Pressure ReadPressure()
         ///  Sea-level pressure
         /// </param>
         /// <returns>
-        ///  Height in meters from the sensor
+        ///  Height in metres from the sensor

```

Comment:
it was correct before. We use US spelling in the docs

Reply:
You're right. Fixed.